src/google/protobuf/compiler/csharp/csharp_enum.cc

```diff
@@ -49,8 +49,8 @@ namespace protobuf {
 namespace compiler {
 namespace csharp {
 
-EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor) :
-    SourceGeneratorBase(descriptor->file()),
+EnumGenerator::EnumGenerator(const EnumDescriptor* descriptor, const Options* file_options) :
```

Comment:
The convention was a copy over from [cpp_generator](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/cpp/cpp_generator.cc#L86).

`options` is just as good.